go/pkg/cas/upload.go

```diff
@@ -139,15 +139,15 @@ func (r *UploadResult) Digest(ps *PathSpec) (*digest.Digest, error) {
 	// TODO(nodir): cache this syscall too.
 	info, err := os.Lstat(ps.Path)
 	if err != nil {
-		return nil, err
+		return nil, errors.WithStack(err)
 	}
 
 	key := makeFSCacheKey(ps.Path, info.Mode(), ps.Exclude)
 	switch val, err, loaded := r.u.fsCache.Load(key); {
 	case !loaded:
 		return nil, nil
 	case err != nil:
-		return nil, err
+		return nil, errors.WithStack(err)
```

Comment:
I think errors.Wrap will appease the lint?

Reply:
this is unfortunate. To appease the linter, I am going to have to wrap the error with info that the error already contains. 

Further, this lint error is incorrect. This error comes from this file. What do you suggest to do?

Reply:
I made https://github.com/bazelbuild/remote-apis-sdks/pull/358 for this.
This linter has some false positives, but I think it's still useful to prevent error where we don't come from.

Reply:
The linter is happy now.